image_explorer/image_explorer.py

```diff
@@ -143,6 +145,8 @@ def student_view_data(self, context=None):
 
         description = self._get_description(xmltree)
         background = self._get_background(xmltree)
+        background_src = self._replace_static_from_url(background['src'])
+        background['src'] = self._make_url_absolute(background_src)
```

Comment:
nit: could you call `_make_url_absolute` from within `_replace_static_from_url` so that we only need one line here?
```python
background['src'] = self._make_url_absolute(background['src'])
```